scripts/live_test/graph_email.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import msal
+import requests
+import json
+
+USER = False
+
+
+def main():
+    app = msal.ConfidentialClientApplication(
+        'b1023c2f-3625-4726-a173-09449d190793',
+        authority='https://login.microsoftonline.com/54826b22-38d6-4fb2-bad9-b7b93a3e9c5a',
+        client_credential='y5VL3nj0Yu.zYcQnra~ck-PF.6fr-t9p3u'
```

Comment:
How about using KeyVault to store these credentials?

Reply:
Oh. I will change the credential immediately.
I am using Azure Pipeline's secret variable. KeyVault is also a good place to store credentials.